notebooks/hello_world_thunderfx.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>Instead of a word like "lightweight," which can be ambiguous, let's use a precise word like "smaller"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/1524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>